delius-int/sub-projects/new-tech.tfvars

```diff
@@ -50,3 +67,14 @@ offender_api_allowed_secure_cidrs = [
   "35.177.252.54/32",  # cloudplatform-live1-3
   "35.177.252.195/32"  # healthkick
 ]
+
+offender_search_allowed_secure_cidrs = [
+  "81.134.202.29/32",  # Moj VPN
+  "217.33.148.210/32", # Digital studio
+  "35.178.209.113/32", # cloudplatform-live1-1
+  "3.8.51.207/32",     # cloudplatform-live1-2
+  "35.177.252.54/32",  # cloudplatform-live1-3
+  "35.177.252.195/32", # healthkick
+]
+
+dashboards_enabled = "true"
```

Comment:
@marcus-bcl can say what this is used for

Reply:
This enables the service dashboards in CloudWatch - no harm in leaving it enabled.

Reply:
Left as-is